docs/user-guide/index.md

```diff
@@ -57,6 +57,10 @@ the different pieces:
 * [Connectors](#connectors)
 * [Formatters](#formatters)
 
+## WSL (Windows Subsystem for Linux) Users
```

Comment:
Maybe make this section more general, and rename it to `Support`?

Cc @sonarwhal/contributors